### ISSUE TITLE
[BUG] pypi desc doesn't render for images without alt

### ISSUE THREAD

**Issue by wasi-master** [wasi-master/pypi-command-line#12]
## Describe the bug
It does not work for images without a alt text because it does not have anything to display

## To Reproduce

1. Use the command `pypi desc wheel_filename`
2. See an error

## Expected behavior
Well it should not error

## Information (please try to complete the following information):

- **OS**             : 10
- **Terminal**       : Windows Terminal
- **Shell**          : powershell
- **OS Version**     : 21H2 (OS Build 19044.1320
- **Python Version** : 3.9.6
- **Package Versions**       :-
  ```ini
  bs4==0.0.1
  click==8.0.1
  click-aliases==1.0.1
  click-help-colors==0.9.1
  humanize==3.12.0
  lxml==4.6.3
  packaging==21.2
  questionary==1.10.0
  requests==2.24.0
  requests-cache==0.8.1
  requests-oauthlib==1.3.0
  requests-toolbelt==0.9.1
  rich==10.12.0
  rich-rst==0.2.3
  shellingham==1.4.0
  thefuzz==0.19.0
  typer==0.4.0
  wheel-filename==1.3.0
  ```

## Additional context

Traceback:
```python
❯ pypi desc wheel-filename
ℹ️ Info: Found shortened name 'desc', using 'description'
Traceback (most recent call last):
  File "C:\Program Files\Python 3.9\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\Python 3.9\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Wasi Master\AppData\Programs\Python38\Scripts\pypi.exe\__main__.py", line 7, in <module>
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\pypi_cli\__main__.py", line 1469, in run
    app()
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\typer\main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\click\core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\click\core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\click\core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\click\core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\typer\main.py", line 500, in wrapper
    return callback(**use_params)  # type: ignore
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\pypi_cli\__main__.py", line 608, in description
    console.print(Panel(description, title=f"Description for {package_name}", border_style="bold magenta"))
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\rich\console.py", line 1594, in print
    extend(render(renderable, render_options))
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\rich\console.py", line 1225, in render
    for render_output in iter_render:
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\rich\panel.py", line 176, in __rich_console__
    lines = console.render_lines(renderable, child_options, style=style)
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\rich\console.py", line 1260, in render_lines
    lines = list(
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\rich\segment.py", line 269, in split_and_crop_lines
    for segment in segments:
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\rich\console.py", line 1225, in render
    for render_output in iter_render:
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\rich\padding.py", line 92, in __rich_console__
    lines = console.render_lines(
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\rich\console.py", line 1260, in render_lines
    lines = list(
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\rich\segment.py", line 269, in split_and_crop_lines
    for segment in segments:
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\rich\console.py", line 1225, in render
    for render_output in iter_render:
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\rich_rst\__init__.py", line 160, in __rich_console__
    document.walkabout(visitor)
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\docutils\nodes.py", line 214, in walkabout
    if child.walkabout(visitor):
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\docutils\nodes.py", line 214, in walkabout
    if child.walkabout(visitor):
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\docutils\nodes.py", line 206, in walkabout
    visitor.dispatch_visit(self)
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\docutils\nodes.py", line 1995, in dispatch_visit
    return method(node)
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\rich_rst\__init__.py", line 107, in visit_image
    Text("🌆 ") + Text(node["alt"], style=Style(link=target or node.get("uri"), color="#6088ff"))
  File "C:\Users\Wasi Master\AppData\Programs\Python38\site-packages\docutils\nodes.py", line 625, in __getitem__
    return self.attributes[key]
KeyError: 'alt'
```

**Comment by wasi-master** [comment]
This is a problem with `rich-rst 0.2.3`. Update to `rich-rst 0.2.4` to fix.

```cmd
pip install -U rich-rst
```
